sdk/eventhub/Azure.Messaging.EventHubs/src/Primitives/EventProcessor.cs

```diff
@@ -1668,34 +1674,56 @@ private async Task RunProcessingAsync(CancellationToken cancellationToken)
                     // Execute the current load balancing cycle.
 
                     Logger.EventProcessorLoadBalancingCycleStart(Identifier, EventHubName, partitionIds?.Length ?? 0, startingOwnedPartitionCount);
+                    TimeSpan remainingTimeUntilNextCycle;
+
+                    try
```

Comment:
For some, there wasn't a try/catch in the main loop so, in the very unlikely case that load balancing or partition health checks threw, the processor would fault.   We want this to be resilient and keep retrying so - inner try/catch!